spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -1020,6 +1020,30 @@ the request enters the JAX-RS method and the timeout information found on the `@
 the moment calculated for the LRA at that time, this new moment becomes the time out moment for the LRA.
 So when multiple participants define a timeout period, the earliest one will trigger the cancellation of the LRA.
 
+Since the specification targets microservices running in different JVM's there are
+potential issues around how timing constraints are handled:
+
+A timeout is a duration which results in some calculated end time. If different systems
+use different timezone or clock granularity then there is potential for different views
+about the current time.  Therefore implementations must be careful to avoid using absolute
+times and be aware that absolute synchronisation between clocks is not possible, and
+therefore, where possible, rely on a duration instead. If an absolute time is required
+then the recommendation is to use UTC which is a universal time standard to which
+all timezones can be canonically converted. This will provide some degree of alignment
+between clocks. If there are edge cases around timing of actions or if local clocks
+cannot be relied upon then there are risks that different parts of the system may
+perform conflicting actions. The implementation must strive to minimise these risks
+but if they do occur then it MUST detect the conflict and force the LRA into
+<<lra-state-model,one of the failure states>>.
+
+Note that time limits are never absolute, they are hints to the system that if the
+time limit is breached then compensation may not be possible and an implementation
+should move the state of the LRA to cancelling. Note that if a time limit is
+breached and the system begins asking some participants to complete before it
```

Comment:
>and the system begins asking some participants to complete before it notices then there is potential for conflicts.

Unclear sentence for me.

Reply:
I agree, it's not clear what the meaning is to me either. I assume it is pointing out that the participant may have made a unilateral decision with prevents a consistent outcome, rather than some confusion in the LRA implementation.

Reply:
You are right, I will reword it (I should have used cancel not close). The sentence was meant to refer to the implementation not noticing that the time limit has passed (for whatever reason). If is then asked to cancel the LRA then that would potentially conflict with the participant that said it will be unable to compensate.